### ISSUE TITLE
Suggestion: Statistics

### ISSUE THREAD

**Issue by GoogleCodeExporter** [andrew867/edcast-reborn#31]
```
Hi   o/

I really miss that on Edcast. The best example is the statistics of the 
SimpleCast. 

I tried to find a software with only this function, but didn't find any. Do you 
know anyone?

I think this will be a great feature to Edcast (Riocast?) Even better if 
supports Icecast 2 and shoutcast.

Regards,
Caio

```

Original issue reported on code.google.com by `china.c...@gmail.com` on 20 Jan 2012 at 12:56

**Comment by GoogleCodeExporter** [comment]
```
I'm not familiar with SimpleCast, nor what "statistics" you require
```

Original comment by `jaroma...@gmail.com` on 22 Jan 2012 at 3:26
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
An example is at: 
http://img.brothersoft.com/screenshots/softimage/s/simplecast-19594-1230625527.j
peg  and a tool is like Radio Toolbox 1.1.1 (from 2005/6), at: 
http://www.radiotoolbox.com/downloads/ 
I guess it helps, best regards,

Gi
```

Original comment by `gr1...@gmail.com` on 13 Jun 2012 at 2:47

**Comment by GoogleCodeExporter** [comment]
```
Oh about the Toolbox - the new Beta version is unstable and for the old version 
1.1.1 to use the Icecast v2 you should have the password of the streamer with 
the statics are not unblocked.... and by the way the password should be of 
yours but I saw that the user as not well configured by some hosts.... you 
should use the user as admin with one of yours passwords.... everything have 
more details........ best regards, Gi 
```

Original comment by `gr1...@gmail.com` on 3 Jul 2012 at 12:29

**Comment by GoogleCodeExporter** [comment]
Original comment by `jaroma...@gmail.com` on 9 Feb 2013 at 2:45
- Added labels: **Milestone-Release4.0**, **Priority-NextRelease**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `jaroma...@gmail.com` on 9 Feb 2013 at 2:55
- Changed state: **NextRelease**

**Comment by GoogleCodeExporter** [comment]
Original comment by `jaroma...@gmail.com` on 21 Dec 2014 at 11:29
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
Original comment by `jaroma...@gmail.com` on 21 Dec 2014 at 11:31
- Added labels: **Priority-Low**, **Type-Other**
- Removed labels: **Priority-NextRelease**, **Type-Enhancement**
